### PR TITLE
7DyD3EvQ: de-duplicate the Compliance Tool DTOs

### PR DESCRIPTION
**Pull request by chrisholmes**
These DTOs became duplicated with the introduction of development mode

### CODE REVIEW

File: src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java [hunk 1]
```diff
@@ -32,10 +34,7 @@
 import static uk.gov.ida.verifyserviceprovider.dto.LevelOfAssurance.LEVEL_1;
 import static uk.gov.ida.verifyserviceprovider.dto.LevelOfAssurance.LEVEL_2;
 import static uk.gov.ida.verifyserviceprovider.dto.NonMatchingScenario.IDENTITY_VERIFIED;
-import static uk.gov.ida.verifyserviceprovider.services.ComplianceToolService.VERIFIED_USER_ON_SERVICE_WITH_NON_MATCH_SETTING_ID;
-import static uk.gov.ida.verifyserviceprovider.services.ComplianceToolService.AUTHENTICATION_FAILED_WITH_NON_MATCH_SETTING_ID;
-import static uk.gov.ida.verifyserviceprovider.services.ComplianceToolService.FRAUDULENT_MATCH_RESPONSE_WITH_NON_MATCH_SETTING_ID;
-import static uk.gov.ida.verifyserviceprovider.services.ComplianceToolService.NO_AUTHENTICATION_CONTEXT_WITH_NON_MATCH_SETTING_ID;
+import static uk.gov.ida.verifyserviceprovider.services.ComplianceToolService.*;
```
Comment (kerrr) [inline src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java]: Star import 🙀

File: src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/ComplianceToolV2InitialisationRequestBuilder.java [hunk 2]
```diff
@@ -20,15 +20,7 @@
...
-            null,
-            singletonList(new MatchingAttribute("Smith", true, LocalDateTime.now().minusDays(30), LocalDateTime.now())),
-            new MatchingAttribute("NOT_SPECIFIED", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()),
-            null,
-            singletonList(new MatchingAddress(true, LocalDateTime.now().minusDays(30), LocalDateTime.now(), "E1 8QS", Arrays.asList("The White Chapel Building" ,"10 Whitechapel High Street"), null, null)),
-            UUID.randomUUID().toString()
-    );
+    private MatchingDataset matchingDataset = new MatchingDataset(new MatchingAttribute("Bob", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()), null, singletonList(new MatchingAttribute("Smith", true, LocalDateTime.now().minusDays(30), LocalDateTime.now())), new MatchingAttribute("NOT_SPECIFIED", true, LocalDateTime.now().minusDays(30), LocalDateTime.now()), null, singletonList(new MatchingAddress(true, LocalDateTime.now().minusDays(30), LocalDateTime.now(), "E1 8QS", Arrays.asList("The White Chapel Building" ,"10 Whitechapel High Street"), null, null)), UUID.randomUUID().toString());
```
Comment (kerrr) [inline src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/ComplianceToolV2InitialisationRequestBuilder.java]: Is this change just from an automated formatting?

Comment (chrisholmes) [reply src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/builders/ComplianceToolV2InitialisationRequestBuilder.java]: Yeah, I was inlining it from the IDE. Looks like I forgot to fix this one after it was inlined.

File: src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java (hunk 1 above)
Comment (chrisholmes) [reply src/acceptance-test/java/uk/gov/ida/verifyserviceprovider/NonMatchingAcceptanceTest.java]: I've tried to disable that so many time in IntelliJ now - I don't know why that's still there :(

**Pull request merged**